pyproject.toml

```diff
@@ -5,6 +5,7 @@ dependencies = ["typing-extensions"]
 
 [project]
 name = "snake-egg"
+requires-python = '>=3.10'
```

Comment:
Please don't limit the library for python 3.10+

Reply:
For python3.9 and below, people can use an older release? 

My reasoning for doing this is that I think we should pin to the acceptance of the match api and other associated features.